src/media.rs

```diff
@@ -132,6 +146,34 @@ mod tests {
     );
   }
 
+  #[test]
+  fn extension_for_content_type() -> Result {
+    let test_cases = vec![
+      ("application/json", "json"),
+      ("image/jpeg", "jpg"),
+      ("image/png", "png"),
+      ("image/svg+xml", "svg"),
+      ("text/html;charset=utf-8", "html"),
+      ("text/plain;charset=utf-8", "txt"),
+      ("text/html", "html"),
+      ("text/plain", "txt"),
```

Comment:
There are inscriptions in `mainnet` with content type `text/plain;charset=utf-8` and others with just `text/plain` (even though `ord` will not create those, at least in its currrent version). This PR also pulls in `accept-header` for lenient content type parsing.